commands/mrHelper.js

```diff
@@ -538,18 +540,30 @@ module.exports = {
             { name: 'alphabetical', value: 'alphabetical' },
             { name: 'level', value: 'level' },
         ))
-    ),
+    )
+    .addSubcommand((subcommand) => subcommand.setName('says').setDescription('Hear what the Hero of Orgrimmar has to say'))
+    .addSubcommand((subcommand) => subcommand.setName('shouts').setDescription('We all know which quote you\'re looking for...')),
```

Comment:
and this will post for all to see. Just to make sure people understand this ahead of time.

Reply:
Updated this text to "Inspire the entire channel with a rallying cry! But we all know which quote you\'re looking for...". How do you feel about that?

Reply:
Hahaha WIN